wpilibj/src/main/java/edu/wpi/first/wpilibj/ADIS16470_IMU.java

```diff
@@ -312,7 +313,7 @@ public ADIS16470_IMU() {
    * @param roll_axis The axis that measures the roll
    */
   public ADIS16470_IMU(IMUAxis yaw_axis, IMUAxis pitch_axis, IMUAxis roll_axis) {
-    this(yaw_axis, pitch_axis, roll_axis, SPI.Port.kOnboardCS0, CalibrationTime._4s);
+    this(yaw_axis, pitch_axis, roll_axis, SPI.Port.kOnboardCS0, CalibrationTime._1s);
```

Comment:
Why change the calibration time?

Reply:
4 seconds doesn't buy you any improvement, but instead increases the window where any erroneous motion of the robot will break the offset estimate.